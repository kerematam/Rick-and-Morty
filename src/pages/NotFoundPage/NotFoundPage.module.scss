.img_container {
  display: flex;
  justify-content: center;
}

.img {
  max-width: 960px;
  min-width: 400px;
  height: 100%;
  width: 100%;
  object-fit: contain;
}
