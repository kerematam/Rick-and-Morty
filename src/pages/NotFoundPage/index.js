import React from 'react'
import NotFoundImg from 'assets/404.jpg'

import styles from './NotFoundPage.module.scss'

const NotFoundPage = () => (
  <div className={styles.img_container}>
    <img alt="404" src={NotFoundImg} className={styles.img} />
  </div>
)

export default NotFoundPage
