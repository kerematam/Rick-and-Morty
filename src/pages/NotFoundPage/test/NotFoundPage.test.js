import React from 'react'
import renderer from 'react-test-renderer'
import NotFoundPage from '../index'

const setup = () => {
  const wrapper = renderer.create(<NotFoundPage />).toJSON()

  return {
    wrapper,
  }
}

describe('NatfoundPage', () => {
  test('should render correctly', () => {
    const { wrapper } = setup()
    expect(wrapper).toMatchSnapshot()
  })
})
