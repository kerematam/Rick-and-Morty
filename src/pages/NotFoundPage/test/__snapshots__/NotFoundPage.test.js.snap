// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NatfoundPage should render correctly 1`] = `
<div
  className="img_container"
>
  <img
    alt="404"
    className="img"
    src="404.jpg"
  />
</div>
`;
